docs/plugins/discovery-ec2.asciidoc

```diff
@@ -1,167 +1,215 @@
 [[discovery-ec2]]
-=== EC2 Discovery Plugin
+=== EC2 discovery plugin
```

Comment:
the other sections on discovery have the same previous heading style. Not sure we should just change it here...

Reply:
Ok, fixed in b3dc74f17e91fe1aa16d2a772f7cfcc403e89eeb